src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,17 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.12.1
+
+This is a bugfix release:
+
+* The default image type for a dedicated gateway node is changed to a smaller one for OpenStack Cloud prepare.
+* `subctl gather` will now use `libreswan` as  a default cable driver if none is specified in `SubmarinerSpec` during installation.
+* Sometimes when Submariner is used to connect onPrem clusters with Public clusters, MTU issues are seen. This was particularly noticed
+when the underlying platform uses `nftables` on the host nodes. This release fixes the MTU issues by explicitly clamping the TCP MSS to a
+fixed value derived from the default interface MTU subtracted with the cable-driver overhead.
+* As part of `subctl uninstall` operation, we now remove the `submariner.io/globalIp` annotation that is added on the Gateway node.
```

Comment:
```suggestion
* As part of `subctl uninstall` operation, we now remove the `submariner.io/globalIp` annotation that is added on the gateway node.
```

Reply:
I am not sure if we should capitalize "gateway" or not (here for e.g we do: https://submariner.io/getting-started/architecture/gateway-engine/), but at least make it consistent (see L11).